content/spin/install.md

```diff
@@ -79,7 +79,9 @@ To install the canary version of spin, you should pass the argument `-v canary`.
 
 {{ startTab "Windows"}}
 
-On Windows systems, you can download <a href="https://github.com/fermyon/spin/releases/latest" class="spin-install" id="spin-install-windows">the Windows binary release of Spin</a>.
+If you want to use WSL (Windows Subsystem for Linux), please follow the instructions for using Linux.
```

Comment:
This might sit more happily in a note/blockquote so that it is set aside from the actual-Windows instructions.  I'd also move it below the actual-Windows instructions (and revert that paragraph).  The reason is that WSL users generally know they are using WSL and will tend to gravitate to the Linux instructions, so this tab should focus on the actual-Windows case as its primary concern, rather than as the "else" part of an if statement _grin_

Reply:
Like this?
<img width="950" alt="image" src="https://user-images.githubusercontent.com/12242001/225756832-6c893cc1-0fb7-464b-b98b-8e3ee5660d84.png">

Reply:
I was thinking inside the Windows tab, but that is probably clearer - thanks!

(I would still drop the reference to "natively.")

Reply:
Going with this, the block thing made it stand out far too much:
<img width="882" alt="image" src="https://user-images.githubusercontent.com/12242001/225843950-1e535d2e-b01a-4a7f-a670-3bb2fe2636ca.png">